# Service to request the parameters for a fiducial array 

---
FiducialArrayInfo info
